app/_src/kubernetes-ingress-controller/production/observability/events.md

```diff
@@ -92,4 +92,42 @@ type: Warning
 {{site.kic_product_name}} creates one Event for each problem with a
 resource, so you may see multiple Events for a single resource with different
 messages. The message describes the reason the resource is invalid. In this
-case, it's because gRPC routes cannot use HTTP methods.
\ No newline at end of file
+case, it's because gRPC routes cannot use HTTP methods.
+
+### Events for cluster scoped resources
+
+Kubernetes Events are namespaced and created in the same namespace as the involved object.
```

Comment:
```suggestion
Kubernetes events are namespaced and created in the same namespace as the involved object.
```

Reply:
👍 I thought that using the name from the API - https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/ - would make sense but perhaps this is better (as it's also not using the code syntax).